src/utils/html_table_extractor.py

```diff
@@ -1,12 +1,12 @@
 import re
 from bs4 import BeautifulSoup
 
-
 class HtmlTableExtractor:
 
     no_th_css = "table > :not(thead):first-child > tr:first-child > :not(th)"
     no_row_header_css = "table tbody tr:nth-child(2) th:not([scope=row]), table tbody tr:first-child *:not(th):first-child"
     two_column_table_css = "td:first-child:nth-last-child(2), td:first-child:nth-last-child(2) ~ td"
+    table_mentions = "the table"
```

Comment:
Might we also want to test for things like "see table" or "in table"?
I know there is no end to potentials and have missed conversations around this, so totally fine with it as it stands as well.

Reply:
Hmm, maybe for the first in-earnest run I should just make it "table" but output the mentions in the report, see what that pulls up.

Reply:
No, that's probably too much. But "see table" and "in table", yes. Good catch.